src/Components/DynamicContent/PromotionBuilder/InputBox.js

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
there is a design conflict between this and checkbox.
In case of checkbox, the code has been split into component and the container, where the container adds the fieldset and legend. In case of InputBox, however, entire component is created within the same file. This may lead to diverging code patterns, which in turn leads to un maintainable code.